app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -5,7 +5,7 @@ extends 'katello/api/v2/common/timestamps'
 
 attributes :content_type
 attributes :docker_upstream_name
-attributes :unprotected, :full_path, :checksum_type, :container_repository_name
+attributes :unprotected, :full_path, :checksum_type, :container_repository_name, :download_policy
```

Comment:
It looks like we are currently always displaying docker-specific attributes here like container_repository_name. I don't really care either way. I'll let @cfouant chime in since she worked on this code.